makefiles/info.inc.mk

```diff
@@ -114,6 +114,70 @@ info-build:
 	@echo ''
 	@echo -e 'MAKEFILE_LIST:$(patsubst %, \n\t%, $(abspath $(MAKEFILE_LIST)))'
 
+define json_string_or_null
+$(or $(1:%="%"), null)
+endef
+
+space :=
+space +=
+comma := ,
```

Comment:
There is already a definition for comma in multiple places, could the be put in common? Also  maybe the following is more exact:

```
comma:= ,
empty:=
space:= $(empty) $(empty)
```?

Reply:
> There is already a definition for comma in multiple places, could the be put in common?

I couldn't find a common one. I think it makes sense to put these in a common location. Do you have a sugestion for which makefile to put them in?

Reply:
> Also maybe the following is more exact:

[This blog](https://www.cmcrossroads.com/article/gnu-make-escaping-walk-wild-side) has both solutions as a suggestion. Your option looks more readable to me, so I can replace my definition.

Reply:
There is a `comma := ,` in `makefiles/arch/mips.inc.mk` and `charCOMMA := ,` in `tests/unittests/Makefile`.

> Do you have a sugestion for which makefile to put them in?

`makefiles/utils/something`

Reply:
> > Also maybe the following is more exact:
> 
> [This blog](https://www.cmcrossroads.com/article/gnu-make-escaping-walk-wild-side) has both solutions as a suggestion. Your option looks more readable to me, so I can replace my definition.

Your choice! I care more of butting them somewhere in common!

Reply:
Done. I'll put the commits in a logical order when squashing.